frontend/src/components/CustomRow.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import DroppableRow from "./DroppableRow";
+import ResultRow from "./ResultRow";
+interface CustomRowProps {
+    turkish: string,
+    englishSelectingWord?: string,
+    english?: string,
+    correctEnglish: string,
+    removeWord: () => void;
+    wordDropped: (word: string) => void;
+    showingResults: boolean,
+    showingAnswers: boolean, 
+    isPairing: boolean
+}
+
+export default function CustomRow({ turkish, englishSelectingWord, english, removeWord, wordDropped, showingResults, showingAnswers, correctEnglish, isPairing }: CustomRowProps) {
+    if (showingResults) {
+      return (<ResultRow
+        key={turkish}
+        showingAnswers={showingAnswers}
+        turkish={turkish}
+        showingResults={showingResults}
```

Comment:
`showingResults` doesn't seem to be a necessary prop for `ResultRow` - if that's the case, let's remove it